spec/helpers/application_helper_spec.rb

```diff
@@ -43,31 +43,50 @@
   describe "social icons+links" do
     context "render if enabled" do
       it "generate rss <link> tag for <head>" do
-        helper.rss_head_link.should eq "<link href=\"http://test.host/monologue/feed\" rel=\"alternate\" title=\"RSS\" type=\"application/rss+xml\" />"
+        helper.rss_head_link.should have_css("link", href: "http://test.host/feed", rel: "alternate",
+          title: "RSS", type: "application/rss+xml")
       end
 
       it "generate rss icon" do
-        helper.rss_icon.should eq "<a class=\"social\" href=\"http://test.host/monologue/feed\" target=\"_blank\"><i class=\"foundicon-rss\"></i></a>"
+        helper.rss_icon.should have_css("a", href: "http://test.host/feed", class: "social",
+          target: "_blank")
+
+        helper.rss_icon.should have_css("i", class: "foundicon-rss")
       end
 
       it "generate github" do
-        helper.github_icon.should eq "<a class=\"social\" href=\"http://github.com/#{Monologue::Config.github_username}\" target=\"_blank\"><i class=\"foundicon-github\"></i></a>"
+        helper.github_icon.should have_css("a",
+          href: "http://github.com/#{Monologue::Config.github_username}", class: "social", target: "_blank")
```

Comment:
Line is too long. [108/80]<br>Align the parameters of a method call if they span more than one line.